doc/strategies/LimitOrder.md

```diff
@@ -0,0 +1,28 @@
+# LimitOrder
+
+The `LimitOrder` contract provides order book style trading functionalities which is similar to what centralized exchange does. Orders can be queried from the off-chain order book system and then be filled on chain. Also, it supports order cancelling and partially fill. The `LimitOrder` contract shares the `AllowanceTarget` design from Tokenlon which allow users to create an order without depositing tokens first. Tokens are transferred between taker and maker only when the order gets filled on chain. This design provides the certainty of price for maker while maintaining filling flexibilities for taker at the same time.
```

Comment:
I don't think the idea for `AllowanceTarget` is `allow users to create an order without depositing tokens first`?  Maybe you are referring to some other contract?

Reply:
Users have to deposit tokens first before creating an order in CEX but not in Tokenlon.

Reply:
Maybe we can just refer the design to DEX or Tokenlon? so reader doesn't have to look into `AllowanceTarget` contract to know what the "design" is as we didn't mention this "design" in the doc before.

Reply:
Actually that's my major purpose. Let me rephrase in a way.

Reply:
Adjusted in ca3f142